widgets/NotificationsWidget.php

```diff
@@ -205,6 +210,7 @@ public function registerAssets()
             'pollSeen' => !!$this->pollSeen,
             'pollInterval' => Html::encode($this->pollInterval),
             'counters' => $this->counters,
+            'tag' => Html::beginTag($this->tag)
         ];
```

Comment:
Instead of the user providing only the tag name, shouldn't we enable him to pass also attributes?
Maybe move the processing in the Javascript, I don't like that we're calling Yii just to get '<' '>'

Reply:
This is a little modification. I have seen that javascript template can be impoved in many ways. I have a few ideas, but now i´m busy with other projects. When i have time i´ll try to do it.